docs/en/getting-started.md

```diff
@@ -1 +1,158 @@
 # Getting Started
+
+Getting up and running with the official `vue-test-utils` package is straight forward.
+Use either `npm` or `yarn` to install the package:
+
+```shell
+# npm
+$ npm install --save vue-test-utils
+
+# yarn
+$ yarn add vue-test-utils
+```
+
+**Notice:** To test Vue components and to make use of the `vue-test-utils` run the specs in a browser with a testrunner like [Karma](https://karma-runner.github.io/1.0/index.html) or node based runners that support virtual DOMs like `jsDOM` (e.g.: [jest](https://facebook.github.io/jest/), [ava](https://github.com/avajs/ava))
```

Comment:
I think the wording could be improved to make it clear that any test runner in node can be used with jsDOM, but that's a minor detail